R/http.R

```diff
@@ -161,7 +164,7 @@ TWIT_paginate_max_id <- function(token, api, params,
       break
     }
     
-    max_id <- id_minus_one(last(get_id(json)))
+    max_id <- max_id(get_id(json))
```

Comment:
I have missed this but json is used on the `is_rate_limit(resp)` before it is generate on `json <- from_js(resp)` (Thanks Rstudio for the static code analysis :). These lines will probably need to be reordered.

Reply:
Ooops, good catch!